one/src/lib.rs

```diff
@@ -70,6 +70,16 @@ struct DaemonOpts {
     )]
     swarm_addresses: Vec<String>,
 
+    /// Extra bootstrap peer addresses to be used in addition to the official bootstrap addresses.
+    /// A best-effort attempt will be made to maintain a connection to these addresses.
+    #[arg(
+        long,
+        use_value_delimiter = true,
+        value_delimiter = ',',
+        env = "CERAMIC_ONE_EXTRA_BOOTSTRAP_ADDRESSES"
```

Comment:
should we call these "bootstrap" addresses or something like "peering" addresses?  We're starting to have more people ask about running ComposeDB with a high availability setup.  For one user to run multiple nodes in an HA cluster, they'll need to make sure all of those nodes are well connected, so I imagine we'll be starting to tell people to use this option soon, so want to be aligned on the best terminology here.

I feel like the hard coded ones are "bootstrap" addresses because they're the main way you discover the rest of the network, but if you're manually setting other addresses to connect to, that's more like a "peering" address (which I think is also the terminology that kubo uses?)

Reply:
Rust-ceramic doesn't distinguish between bootstrap vs peering. I don't have a preference for which term we use, however we should only use one.

Reply:
I'm also cool either way, if you have a preference for "bootstrap" or "peering", @stbrody.

Reply:
Okay, if we feel like we want to keep the same term for both the hard-coded and user-provided peers then I don't feel strongly which one we use.  Fine to keep things as-is